Library/Formula/crosstool-ng.rb

```diff
@@ -42,7 +42,7 @@ def install
 
     args << "--with-grep=ggrep" if build.with? "grep"
 
-    args << "--with-make=gmake" if build.with? "make"
+    args << "--with-make=#{HOMEBREW_PREFIX}/bin/gmake" if build.with? "make"
```

Comment:
Shouldn't do. The line will only be "activated" if someone passes `--with-make`, during which `dupes` will be tapped if it isn't already and it'll be a known formula. The full path for the dep is only necessary because otherwise `brew` doesn't know what `make` formula you're asking it to go find.